crates/oxc_wasm/src/lib.rs

```diff
@@ -188,7 +188,7 @@ impl Oxc {
 
             let semantic = Rc::new(semantic_ret.semantic);
             let lint_ctx = LintContext::new(&semantic);
-            let linter_ret = Linter::new().run(lint_ctx);
+            let linter_ret = Linter::new().run(lint_ctx, vec![Some("index.ts".to_owned())]);
```

Comment:
we don't really need anything in this vec now, but eventually it might be nice to have trustfall queries running in wasm too.